tests/module/checks/test_git.py

```diff
@@ -0,0 +1,75 @@
+from pkgcheck.checks import git as git_mod
+from .. import misc
+from pkgcheck.git import _GitCommit
+
+def mk_commit(
```

Comment:
@radhermit if you've got a preferred style for this sort of indentation, let me know.  I'm following a rough version of google norms- frankly more out of habit then preference.

Reply:
It mostly looks fine, although I usually try not to put the closing parenthesis on the same indent level as the function-scoped code.